prover/src/constraints/composition_poly.rs

```diff
@@ -67,11 +60,29 @@ impl<E: FieldElement> CompositionPoly<E> {
         self.column_len() - 1
     }
 
-    /// Returns evaluations of all composition polynomial columns at point z^m, where m is
-    /// the number of column polynomials.
+    // LOW-DEGREE EXTENSION
+    // --------------------------------------------------------------------------------------------
+    /// Evaluates the columns of the composition polynomial over the specified LDE domain and
+    /// returns the result.
+    pub fn evaluate<B>(&self, domain: &StarkDomain<B>) -> ColMatrix<E>
+    where
+        B: StarkField,
+        E: FieldElement<BaseField = B>,
+    {
+        assert_eq!(
+            self.column_len(),
+            domain.trace_length(),
+            "inconsistent trace domain size; expected {}, but received {}",
+            self.column_len(),
+            domain.trace_length()
+        );
+
+        self.data.evaluate_columns_over(domain)
+    }
```

Comment:
Where is this method being used? (I couldn't find upon a quick search).

Reply:
I agree, removed it now